Services/CompanyService/Database/Migrations/20200823110601_NewDbFileModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.CompanyService.Database.Migrations
+{
+    public partial class NewDbFileModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
The name of the migration should be NewDbPosition model, isn't it?

Reply:
fixed